src/util/filter.js

```diff
@@ -6,8 +6,8 @@ const isValidCrime = crime => offenses.includes(crime)
 const defaults = {
   crime: 'violent-crime',
   place: 'united-states',
-  since: 2004,
-  until: 2014,
+  since: 2005,
+  until: 2015,
```

Comment:
Same point as above, perhaps we should be using `MAX_YEAR` in this util, and create the default `since` by subtracting 10 from it?